docs/documentation/helpcenter.md

```diff
@@ -148,6 +152,32 @@ with modals, etc. should never go in the help center documentation.
 In such cases, you may be able to fix the problem by adding text in-app,
 where the user will see it as they are interacting with the feature.
 
+### Redirecting an existing article
+
+From time to time, there is a reason to rename articles in the help center or
+REST API documentation. This breaks incoming links, including from other
+branches of the repository that haven't been rebased, and more importantly
+from previous versions of Zulip.
+
+To make it easier for contributors make these changes, there is only one place
+where you need to add a redirect: `zerver/lib/documentation_redirects.py`.
+
+Once you've renamed the file (e.g. `foo.md` is now `bar.md`) in your branch,
```

Comment:
We could spell this `git mv path/to/foo.md path/to/bar.md`? Not sure if it's helpful.

Reply:
Updated for that suggestion.